<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1001</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>981</width>
       <height>621</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(195, 190, 180);
border-radius:30px
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>981</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,210);
</string>
      </property>
      <widget class="QWidget" name="horizontalWidget_5" native="true">
       <property name="geometry">
        <rect>
         <x>990</x>
         <y>0</y>
         <width>81</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgba(0,0,0,0);
	color:rgb(255,255,255);
	border-radius:1px;
	font-size:18px;
	font-family:dripicons-v2;
}

QPushButton:hover{
	background-color:rgb(49,48,53)
}

QPushButton#cerrar:hover{
	background-color:rgb(232,17,25)
}

QpushButton:pressed{
	padding-top:5px;
 	padding-left:5px;
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>141</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(255,255,255,255)</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;MATERIAL&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalWidget_6" native="true">
       <property name="geometry">
        <rect>
         <x>930</x>
         <y>0</y>
         <width>51</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgba(0,0,0,0);
	color:rgb(255,255,255);
	border-radius:1px;
	font-size:18px;
	font-family:dripicons-v2;
}

QPushButton:hover{
	background-color:rgb(49,48,53)
}

QPushButton#cerrar:hover{
	background-color:rgb(232,17,25)
}

QpushButton:pressed{
	padding-top:5px;
 	padding-left:5px;
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnCerrar">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnInsertarMaterial">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>440</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:1.5px solid #7c6e59;
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Insertar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>341</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);
border-radius:30px</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblMaterial">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>110</y>
        <width>511</width>
        <height>461</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border: 3px solid  #000000;
background-color: rgb(199, 194, 184);
</string>
      </property>
      <column>
       <property name="text">
        <string>Id - Material</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Id - Servicio</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Descripcion</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stock</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Und. Medida</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Frente</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>500</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/newPrefix/Marohuma_icono.png);
background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Imagenes/QT_imagenes.qrc">:/newPrefix/Marohuma_icono.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNombreMaterial">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
border:1.5px solid #7c6e59;
border-radius:15px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
      <property name="placeholderText">
       <string>Ingrese Nombre del Material</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>Unidad Medida:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>Descripcion:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>Frente:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtFrenteMaterial">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>360</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
border:1.5px solid #7c6e59;
border-radius:15px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
      <property name="placeholderText">
       <string>Ingrese el Frente</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cboUndMedida">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>310</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 190, 180);
border:1.5px solid #7c6e59;
border-radius:10px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
      <item>
       <property name="text">
        <string>--Seleccionar--</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mtrs</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>m2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>m3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Millar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bolsa</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>ID Servicio:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cboServicio">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 190, 180);
border:1.5px solid #7c6e59;
border-radius:10px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
      <item>
       <property name="text">
        <string>--Selecionar--</string>
       </property>
      </item>
     </widget>
     <widget class="QPlainTextEdit" name="pteDescripcion">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>201</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 190, 180);
border:1.5px solid #7c6e59;
border-radius:15px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cboListar">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:1.5px solid #7c6e59;
border-radius:10px;</string>
      </property>
      <item>
       <property name="text">
        <string>--Todos--</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 190, 180);</string>
      </property>
      <property name="text">
       <string>Listar:</string>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>widget_3</zorder>
     <zorder>btnInsertarMaterial</zorder>
     <zorder>tblMaterial</zorder>
     <zorder>label_6</zorder>
     <zorder>txtNombreMaterial</zorder>
     <zorder>label_17</zorder>
     <zorder>label_18</zorder>
     <zorder>label_19</zorder>
     <zorder>txtFrenteMaterial</zorder>
     <zorder>cboUndMedida</zorder>
     <zorder>label_20</zorder>
     <zorder>label_21</zorder>
     <zorder>cboServicio</zorder>
     <zorder>pteDescripcion</zorder>
     <zorder>cboListar</zorder>
     <zorder>label_7</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Imagenes/QT_imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
